pandas/io/clipboard.py

```diff
@@ -74,6 +74,12 @@ def to_clipboard(obj, excel=None, sep=None, **kwargs):  # pragma: no cover
       - Windows:
       - OS X:
     """
+    encoding  = kwargs.get('encoding')
+
+    #testing if an invalid encoding is passed to clipboard
+    if encoding is not None and encoding is not 'utf-8':
```

Comment:
encoding not None and encoding.lower().replace('-','') != 'utf8'
would be correct?, if we are replacing then utf-8 wont have the "-" either